tests/e2e.test.js

```diff
@@ -295,17 +296,34 @@ describe('Ordering', () => {
     }
   });
 
-  it.skip('orders grid items by color', async () => {
-    // Items are ordered using the CSS property `order`...
-    await expect(page).toClick('#order-color');
+  it('orders grid items by color', async () => {
```

Comment:
The tests currently assume the tests are executed in order, every tests that tests something related to color ordering should click the `#order-color` button first.

Reply:
no, the test does not assume any order. The test bypasses the need to click on the order-color: the click on the order color does nothing testable, since the css order property does not change the DOM elements order.
Notice that the test takes the grid cards and reads the --style-color attribute, and makes sure that the value assigned during build is the same as the index of the corresponding hex in the sortByColor call in the unit test.

There is no need for any previous test to be executed.

Reply:
In that case, as per one of my other comments, it's not actually testing if the `#order-color` button does it job, which is what this (or at the very least one of these tests) should do.

Reply:
See also #82:

> We should add an integration test verifying that the "Order by color" button changes the ordering of the icons.

Reply:
there is no way to test that. It's impossible since CSS does not change the order of the DOM elements.
I even tried downloading the page HTML after ordering, and it gets downloaded in alpha sort order.

We can:
- either close this as won't add.
- Leave the test as is, to make sure --order-color is being generated as expected
- Implement #86 by moving around DOM elements, and try again.

Reply:
I agree that it is likely impossible to test this with our current setup given that the DOM order doesn't change.

If we want to test that `--order-color` is being generated as expected, then I don't think it should be an end-to-end test. Again, you're not testing something the users sees. For example, I can easily break the website without breaking this test by removing [this line of CSS](https://github.com/simple-icons/simple-icons-website/blob/00139387a7133ce1eadd86eb70a0e28e8f35dac4/public/stylesheet.css#L704). That's what I mean when [I say](https://github.com/simple-icons/simple-icons-website/pull/110#discussion_r706635186) that "it gives a false sense that everything is okay".

Implementing the DOM-reordering suggestion for #86 would indeed make this actually end-to-end testable. Having this tested would be the preferred option. I'd personally be against closing it as "won't add" because we can't think of a solution now and we _want_  to reach 0 open issues, but if you think that makes sense, again, feel free to close those issues.

Reply:
I'm going to remove the test, then.
However, #86 implementation is not going to be done anytime soon. It's not extremely hard, but the performance hit is going to be painful.

Reply:
Test is removed. shall we merge the algorithm?

Reply:
I'm not saying this shall not be merged unless the test is disabled. I'm only sharing my thoughts, [I'm not commanding you to do anything](https://github.com/simple-icons/simple-icons-website/pull/110/commits/bbe6e645138b026447eb2fac7db36fae2a182071). I'm simply sharing my believe that in this case no test may be better than some test.

If you think my reviews are to fine grained, don't ask me for a review. If you think my reviews are slowing down development to much, don't ask for my reviews (as I'm of the opinion that for a project like this, quality goes over quantity)). If you think this test is slowing down the bugfix getting merged, then I refer you to [this comment](https://github.com/simple-icons/simple-icons-website/issues/82#issuecomment-914564965).